applications/cmd_line/src/svk_quantify.cc

```diff
@@ -279,12 +295,9 @@ int main (int argc, char** argv)
             quantMets->LimitToSelectedVolume();
         }    
         quantMets->SetInput( reader->GetOutput() ); 
-        quantMets->LimitToSelectedVolume();    
```

Comment:
Hi,

Shouldn't this be conditional based on the -b flag, rather than deleted?

-jason